docs_src/examples/use_cases/example_first_order_resnet.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, C_in=1, C_hid=5, input_dim=(28, 28), output_dim=10):
     def forward(self, x):
         residual = self.shortcut(x)
         x = self.conv2(F.relu(self.conv1(x)))
-        x += residual
+        x = x + residual  # don't use: x += residual
```

Comment:
will the inplace operation cause a problem?

Reply:
From torch 1.9.0 onwards it throws the following error
```
RuntimeError: Output 0 of BackwardHookFunctionBackward is a view and is being modified inplace. This view was created inside a custom Function (or because an input was returned as-is) and the autograd logic to handle view+inplace would override the custom backward associated with the custom Function, leading to incorrect gradients. This behavior is forbidden. You can remove this warning by cloning the output of the custom Function.
```
Seems like using a backward hook in combination with inplace and views does not work.